_posts/2019-02-05-configurable-builds-part-1.md

```diff
@@ -202,28 +201,27 @@ and built with:
 
     $ bazel build //my:binary --platforms=//platforms:pixel3 --define MYFEATURE=1
 
-This is more concise and correct than before. Now it doesn't matter what the
-CPU is. Any platform with the `:android`
-[`constraint_value`](https://docs.bazel.build/versions/master/be/platform.html#constraint_value)
-is an Android platform. So it's easy to express *exactly* what you want and
-have `select`, rules, and toolchains understand it the same way. `MYFEATURE=1`,
-which isn't a platform property, remains as before.
+This is a significant improvement over before. It's more concise and you no
+longer have to care what the CPU is. *Any* platform tagged with `:android` is an
```

Comment:
Remove 'tagged', since that has a different meaning in Bazel. Possibly "any platform with an `:android` constraint" is better?

Reply:
Done.